resources/js/components/MegaFilter.vue

```diff
@@ -182,6 +183,7 @@
     }
 
     .nova-mega-filter {
+        overflow: visible;
```

Comment:
I this needed? this would probably create an ugly white scroll bar on windows browsers...

Reply:
This css is an update for this PR https://github.com/dcasia/nova-mega-filter/pull/99
Because Nova Card has the default class ` overflow-hidden` which blocks the dropdown display.

Reply:
I think the situation you mentioned only happens when we use `overflow: scroll;`. 
I believe `overflow: visible;` works fine since we don't fix the height.